src/app/app/app-layout/editor-footer/editor-footer.component.ts

```diff
@@ -16,17 +16,22 @@
  */
 
 import { Component, Inject } from '@angular/core';
-import { EDITOR_FOOTER_SERVICE_TOKEN, EditorFooterService } from '../../../services/editor-footer.service.interface';
+import { EDITOR_FOOTER_SERVICE_TOKEN, EditorFooterService } from './editor-footer.service.interface';
+import { Store } from '@ngrx/store';
+import { AmaState } from 'ama-sdk/src/public_api';
+import { selectToolbarLogsVisibility } from '../../../../app/store/selectors/app.selectors';
 
 @Component({
-    selector: 'amasdk-editor-footer',
+    selector: 'ama-editor-footer',
     templateUrl: './editor-footer.component.html'
 })
 export class EditorFooterComponent {
 
     showConsole = false;
 
-    constructor(@Inject(EDITOR_FOOTER_SERVICE_TOKEN) public editorFooterService: EditorFooterService) {}
+    constructor(@Inject(EDITOR_FOOTER_SERVICE_TOKEN) public editorFooterService: EditorFooterService, private store: Store<AmaState>) {
+         this.store.select(selectToolbarLogsVisibility).subscribe(visibily => this.showConsole = visibily);
```

Comment:
Subscription without unsubscribe in ngOnDestroy!!!!
Instead of this why not transforming `showConsole` into this:

showConsole$ = this.store.select(selectToolbarLogsVisibility);

This we can remove this line from toggleConsole also:
```
this.showConsole = !this.showConsole;
```

and we can use the async pipe in the template so don't need to take care of unsubscribing.